e2e/tests/email-campaign.test.ts

```diff
@@ -1,10 +1,10 @@
-import test, { expect } from '@playwright/test';
-import { Page } from 'playwright';
+import test, {expect} from '@playwright/test';
```

Comment:
Can you run `npm run lint` at project root to get rid of these formatting diffs?

Reply:
this command didn't work, but i had to run `npx prettier --single-quote --trailing-comma all --write e2e/*` to get the right formatting. perhaps something wrong with my prettier config that i had to do it manually